platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -315,6 +317,7 @@ function WorkList({ history, data: studies, isLoadingData, dataSource }) {
                   disabled={false}
                   endIcon={<Icon name="launch-arrow" />} // launch-arrow | launch-info
                   className={classnames('font-bold', { 'ml-2': !isFirst })}
+                  onClick={() => {}}
```

Comment:
(fix console error) -- Adding an empty `onClick` because `Button` expects this as a required prop.